New depad-utils and new + updated Picard versions and updated snptest version.

pneerincx opened the pull request:
* New depad-utils version v17.04.1
* New picard version 2.9.2
* Fixed various picard versions: added checksums.
* Fixed snptest version: removed debug statements.

mmterpstra commented:
Needs testing and all easybuild files depending on picard should be fixed and tested (some legency tool might need older versions/do not support the new picard syntax). Also at peregrene cluster I need to deploy tools towards foss2016a so why not keep the toolchain version is the name.

pneerincx commented:
@mmterpstra: Ok I've reverted the commit that deleted the problematic Picard versions and added checksums for the sources to those. I understand you will have problems if the toolchain version for a dependency is not part of the EasyConfig file name and you have the same version of Picard on different clusters with different versions of the toolchain for that dependency... but I'm not sure how this may interfere with EasyBuild's robot function, which expects filenames in a certain format. I've update EasyBuild to our clusters to a newer version that allows strict checking of filenames for the config files to make sure they are compliant.
